universal-application-tool-0.0.1/app/views/questiontypes/FileUploadQuestionRenderer.java

```diff
@@ -25,6 +22,8 @@
 public class FileUploadQuestionRenderer extends ApplicantQuestionRenderer {
   private static final String IMAGES_AND_PDF = "image/*,.pdf";
 
+  @Inject FileUploadStrategy fileUploadStrategy;
```

Comment:
suggestion (blocking): inject the constructor instead and make this member `private final`

Reply:
Doing this ended up being a little convoluted and required changing a couple more files. Can you let me know how it looks to you? I'm not sure I did exactly what you had in mind.

Reply:
Ah, I was confused and thought this class is instantiated by guice but turns out it's initialized by application code. Your changes look right.